core/repository/block/pg.ts

```diff
@@ -59,7 +59,8 @@ class BlockPGRepo implements IBlockPGRepository {
             fee: rawBlock.fee,
             payloadHash: rawBlock.payload_hash,
             generatorPublicKey: rawBlock.generator_public_key,
-            signature: rawBlock.signature
+            signature: rawBlock.signature,
+            transactions: []
```

Comment:
when we will push transactions to block?

Reply:
after deserialize